broker/backend/kinesis/processor.go

```
@@ -59,37 +58,23 @@ func (p *processor) consumeRecords() {
 		case <-p.quit:
 			return
 		default:
-			record, err := p.kinsumer.Next()
-			if err != nil {
-				panic(err) // TODO: manage error
-			}
+			record, _ := p.kinsumer.Next()
```

How is the error being handled now? Or is it safe to ignore, as this change seems to be asserting? Would like to see a comment here about why it is okay to use `_` to ignore the error.

---

Yes, sorry, I meant to add some extra logic in the kinesis backend to launch a new processor under these circumstances. But I realize it's enough for now to panic and let the process die, eventually the service manager will restart the application and things should work again. Perhaps not idea but good enough.